go get -u github.com/Azure/azure-amqp-common-go/... fails

### Expected Behavior
`$ go get -u github.com/Azure/azure-amqp-common-go/...` should run without failing.

### Actual Behavior
It outputs this error message:
```
# github.com/Azure/azure-amqp-common-go/aad
../../src/github.com/Azure/azure-amqp-common-go/aad/jwt.go:203:44: cannot use token.ExpiresOn (type json.Number) as type string in argument to strconv.ParseInt
../../src/github.com/Azure/azure-amqp-common-go/aad/jwt.go:216:69: cannot use token.ExpiresOn (type json.Number) as type string in argument to auth.NewToken
```

### Environment
- OS: Debian GNU/Linux
- Go version: go1.11rc2 linux/amd64
- Version of Library: dead23a10516c64bf754a2aabc7063304adcf736

--- a/aad/jwt.go
+++ b/aad/jwt.go
@@ -200,7 +200,7 @@ func (c *TokenProviderConfiguration) NewServicePrincipalToken() (*adal.ServicePr
 // GetToken gets a CBS JWT
 func (t *TokenProvider) GetToken(audience string) (*auth.Token, error) {
 	token := t.tokenProvider.Token()
-	expireTicks, err := strconv.ParseInt(token.ExpiresOn, 10, 64)
+	expireTicks, err := strconv.ParseInt(string(token.ExpiresOn), 10, 64)
 	if err != nil {
 		return nil, err
 	}
@@ -213,7 +213,7 @@ func (t *TokenProvider) GetToken(audience string) (*auth.Token, error) {
 		token = t.tokenProvider.Token()
 	}
 
-	return auth.NewToken(auth.CBSTokenTypeJWT, token.AccessToken, token.ExpiresOn), nil
+	return auth.NewToken(auth.CBSTokenTypeJWT, token.AccessToken, string(token.ExpiresOn)), nil
 }
 
 func decodePkcs12(pkcs []byte, password string) (*x509.Certificate, *rsa.PrivateKey, error) {

